app/forms/decidim/anonymous_codes/admin/code_group_form.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      class CodeGroupForm < Decidim::Form
+        include TranslatableAttributes
+
+        translatable_attribute :title, String
+        attribute :expires_at, Decidim::Attributes::LocalizedDate
+        attribute :active, Boolean, default: true
+        attribute :max_reuses, Integer, default: 1
+        attribute :resource_id, Integer
+
+        validates :title, translatable_presence: true
+        validates :expires_at, date: { after: Date.current }, if: ->(form) { form.expires_at.present? && form.active }
```

Comment:
```suggestion
        validates :expires_at, date: { after: Time.current }, if: ->(form) { form.expires_at.present? && form.active }
```
An here the full time

Reply:
Thanks, done.